src/js/store/factory/Scale.ts

```diff
@@ -16,7 +16,9 @@ export type LyraScale = {
    */
   _origName: string;
   _domain: any[],
-  _range: any[]
+  _range: any[],
+  _manual: boolean,
+  _domainArray: any[]
```

Comment:
introduced this so we don't need to override _domain is the user switches to manual. That way if they toggle the manual check the _domain value with fields isn't lost. But we resolve this in the exports.ts file.

Reply:
is `_manualDomain` a clearer variable name for `_domainArray`?